Resfeber/Model Controller/DestinationController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  DestinationController.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 11/7/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class DestinationController {
+    
+    static func createDestination(destination: Destination) {
+        
```

Comment:
Probably good to put a `// TODO:` here so that you know to come back to it

Reply:
Would you suggest adding a build script phrase to highlight the TODO as a warning?

Wondering what's a best practice in industry 

Example: 
```
TAGS="TODO:|FIXME:"
ERRORTAG="ERROR:"
find "${SRCROOT}" \( -name "*.h" -or -name "*.m" -or -name "*.swift" \) -print0 | xargs -0 egrep --with-filename --line-number --only-matching "($TAGS).*\$|($ERRORTAG).*\$" | perl -p -e "s/($TAGS)/ warning: \$1/" | perl -p -e "s/($ERRORTAG)/ error: \$1/"

```